site/docs/components/skip-link/accessibility.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+## Best practices
```

Comment:
Do these best practices make sense for a component? All of this is out of the box behaviour.